### PR TITLE
drivers: npmx: shell: rename buck set to buck status power set

### PR DESCRIPTION
**Pull request by masz-nordic**

### CODE REVIEW

**Comment by wilu-nordic** [comment]
DNM until
 * https://github.com/NordicSemiconductor/npmx-zephyr/pull/59

**Review by wilu-nordic — APPROVED** [review]

File: drivers/npmx/npmx_shell.c [hunk 1]
```diff
@@ -3008,6 +3008,8 @@ SHELL_STATIC_SUBCMD_SET_CREATE(sub_buck_gpio,
 
 /* Creating subcommands (level 4 command) array for command "buck status power". */
 SHELL_STATIC_SUBCMD_SET_CREATE(sub_buck_status_power,
+			       SHELL_CMD(set, NULL, "Set buck power status",
+					 cmd_buck_status_power_set),
```
Comment (magp-nordic) [inline drivers/npmx/npmx_shell.c:3012]: `set` command should be after `get` command

Comment (masz-nordic) [reply drivers/npmx/npmx_shell.c]: fixed

**Review by wilu-nordic — APPROVED** [review]

**Review by jaz1-nordic — APPROVED** [review]

**Comment by nordic-segl** [comment]
First attempt to align test repo to changes from this PR:
https://projecttools.nordicsemi.no/bitbucket/projects/NCS-TEST/repos/test-sdk-npmx/pull-requests/162/overview
Waiting for results
https://jenkins-ncs.nordicsemi.no/job/HCS/job/NPMX/job/npmx_zephyr_integration/view/change-requests/job/PR-162/
https://jenkins-ncs.nordicsemi.no/job/HCS/job/NPMX/job/npmx_zephyr_develop_samples/view/change-requests/job/PR-162/

**Pull request merged**